buku.py

```diff
@@ -67,6 +67,16 @@
 logdbg = logger.debug
 logerr = logger.error
 
+def read_input(msg):
```

Comment:
Will do so. I know this is more or less personal preference, but making function name shorter in the name of 80 chars length doesn't make it more readable. Would recommend raymond hettinger "Beyond PEP8" talk that he gave during one of the Pycon.